app/move/middleware.js

```diff
@@ -29,8 +29,11 @@ module.exports = {
 
     try {
       const move = await req.services.move.getByIdWithEvents(moveId)
+      const journeys = await req.services.journey.getByMoveId(moveId)
+
       populateResources(move.timeline_events, req)
       req.move = move
+      req.journeys = journeys
```

Comment:
I think by having this in `setMoveWithEvents` it won't work on tabs which only use the `setMove` middleware (specifically the details tab).

Reply:
Ahh yes, the details tab does not show the journeys, ill add journeys to `setMove`